### PR TITLE
Update test data bucket

### PR DESCRIPTION
**Pull request by surajkota**
### Testing
PR build should pass

### CODE REVIEW

File: test/e2e/bootstrap_resources.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 from acktest.resources import read_bootstrap_config
 from e2e import bootstrap_directory
 
-SAGEMAKER_SOURCE_DATA_BUCKET = "source-data-bucket-718865417152-us-west-2"
+SAGEMAKER_SOURCE_DATA_BUCKET = "source-data-bucket-592697580195-us-west-2"
```
Comment (ryansteakley) [inline test/e2e/bootstrap_resources.py:21]: Still getting the same error when using this bucket. `botocore.exceptions.ClientError: An error occurred (403) when calling the HeadObject operation: Forbidden
`

Comment (surajkota) [reply test/e2e/bootstrap_resources.py:21]: try now

Comment (ryansteakley) [reply test/e2e/bootstrap_resources.py:21]: Successfully created and copied data from that bucket on my end but looks like some tests are erroring out.

Comment (ryansteakley) [reply test/e2e/bootstrap_resources.py:21]: Looked into it and looks like the s3 bucket you are providing doesn't have validation data for training jobs under `s3://$SAGEMAKER_DATA_BUCKET/sagemaker/training/validation`

Comment (surajkota) [reply test/e2e/bootstrap_resources.py:21]: checking ... maybe the bucket I am copying from doesn't have the latest copy then

**Review by ryansteakley — APPROVED** [review]

**Review by mbaijal — APPROVED** [review]

**Pull request merged**